src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -166,10 +166,10 @@
       [:div {}
        [comps/Button {:text "Sign In"
                       :href (str "/service/login?path="
-                                 (js/encodeURIComponent (let [hash (nav/get-hash-value)]
+                                 (js/encodeURIComponent (js/decodeURIComponent (let [hash (nav/get-hash-value)]
```

Comment:
@dmohs - I'm pretty sure that going through the google sign in process, the original callback url that is part of the overall request gets encoded. By the time that url fragment gets to the orchestration layer, it has been encoded.